cmd/frontend/main.go

```diff
@@ -80,8 +80,8 @@ func main() {
 		logrus.Fatalf("Failed to start monitor: %v", err)
 	}
 
-	configWatcher := make(chan *configuration.Config)
-	configurationWatcher := configuration.NewWatcher(config.ConfigMapName, config.Namespace, configWatcher)
+	configWatcher := make(chan *configuration.OperatorConfig)
+	configurationWatcher := configuration.NewOperatorWatcher(config.ConfigMapName, config.Namespace, configWatcher)
```

Comment:
Not sure if it should be called like this since the config is part of Meridio, and meridio is watching the configmap, not the operator